packages/shared/src/utils/currency/currency.ts

```diff
@@ -42,13 +42,15 @@ const crypto_currencies_display_order = [
     'USDK',
 ];
 
-export const reorderCurrencies = (list: Array<{ value: string; type: string }>, type = 'fiat') => {
+export const reorderCurrencies = <T>(list: Array<T>, type = 'fiat') => {
```

Comment:
```suggestion
export const reorderCurrencies = <T extends { value: string; type: string }>(list: Array<T>, type = 'fiat') => {
```
this will help you remove both // @ts-expect-errors below :)